sections/telemetry_system.tex

```diff
@@ -137,6 +137,21 @@ \subsubsection{Magnetometer}
 This magnetometer was selected because it has an \gls{i2c} interface and can be supplied at 3.3V.
 \cite{lsm6dso32-datasheet}
 
+\section{Board Design}
```

Comment:
I really like this section, it gives good reasoning for the change. Maybe put it as the first section following the "Telemetry System" chapter header? Then people understand the reasoning as they read the rest of the board design stuff.